src/settings_key_variables.cpp

```diff
@@ -31,32 +31,32 @@ std::map<Thumbstick, WORD> THUMBSTICK_KEYS = {
 /**
  * A std::map to map the virtual key codes to corresponding key names
  */
-const std::map<UINT, const char *> vk_maps = {{VK_LBUTTON, "LMButton"},
```

Comment:
Button names are constants.
So why not `const`?

Reply:
Because std::map is ineligible for a const expression.

Reply:
Also const pointer means that the address the pointer is pointing to will always be same so the map is immutable during runtime so map does not require a const.